test/test_transforms_v2.py

```diff
@@ -3099,7 +3132,7 @@ def test_transform_bounding_boxes_correctness(self, output_size, format, dtype,
 
         expected = self._reference_crop_bounding_boxes(bounding_boxes, **params)
 
-        assert_equal(actual, expected)
+        assert_equal(actual, expected, atol=1, rtol=0)
```

Comment:
If we can't fix the differences by "fixing" our reference test function, then increasing the tol here is OK, but we should be careful not to do that unconditionally:

- we only need to increase the tol for rotated formats IIUC
- setting `atol` to 1 for floating dtypes seems too high: some of the errors we have are e.g.
```
E   Mismatched elements: 3 / 5 (60.0%)
E   Greatest absolute difference: 2.1457672119140625e-06 at index (0, 1)
E   Greatest relative difference: 6.130767360446043e-07 at index (0, 1)
```

so we can probably set atol to something like 1e-5 in such cases, which is stricter.